src/frontend/admin/src/hooks/useResources/index.tsx

```diff
@@ -9,6 +9,7 @@ import { HttpError } from "@/services/http/HttpError";
 export interface ResourcesQuery {
   id?: string;
   query?: string;
+  page?: number;
```

Comment:
Are you sure that all GET routes could handle `page` query params? If so, it's fine!